### ISSUE TITLE
Does vsphere support still evolve?

### ISSUE THREAD

**Issue by stainboy** [kubernetes/kops#6531]
I have read all vsphere related documents of kops and found the last update was in April 2018 which was 10 months ago. And the document indicated me to download the kops binary from google drive instead of regular Github release. Furthermore, since vsphere is typically On-Premise setup, which makes it very different from AWS/GCE environment. There is almost no prerequisites or precondition (e.g. how the network and/or storage is configured in vsphere) mentioned before running kops against vsphere. I understood it is in the alpha stage to support vsphere, but I found nothing about the roadmap for this topic.

Put everything together, could someone kindly answer me questions below?
- Does vsphere support still actively evolve?
- What is the position of vsphere support? (on an equal footing to AWS?)
- What is the approximate time it comes GA (e.g. single binary works for both AWS/vsphere, with comprehensive documentation)

**Comment by stainboy** [comment]
Let me explain the intention of the question a little bit.

Back to 2015, I was involved in a large enterprise product development, which extremely embraced container and Kubernetes technology. Under my leadership, a small team of ten started to setup Kubernetes on top of vsphere, and build CI/CD pipelines on top of k8s, which severed for the entire develop team.

There was no efficient tools to create and maintain k8s at the age. So we wrote some bash scripts to bootstrap hundreds of CoreOS instance by interacting with vsphere API. It was a success. We got the working k8s cluster eventually.

The problem was, in the next several years, it took us tremendous effort to maintain the k8s cluster, including configuration changes to specific nodes, k8s version upgrading, etcd data backup and so on. Not mention to keep the cluster status same as the stored status. It's like a nightmare to maintain everything.

In the past several years, a lot of automation tools have been created, such as `kubeadm`, `bootkube`, `garndener` from SAP and etc. They just cannot be simply used to replace the one we "created". Until I found vsphere support by kops, which looks exactly like the one I expected.

In conclusion, I like the project very much and I hope it can evolve and serve more people like me. Thanks.
